### DISCUSSION TITLE
onProgress don't show all progress but just the first added

### DISCUSSION THREAD

**Discussion by nampoinaandriamalala** [norkunas/youtube-dl-php#172] [Q&A]
Hi all!

I tested to download many url simultaneously. The download run but i don't have the others progress. 
Is this possible to show others ?

**Comment by norkunas** [comment]
Hi.
With `simultaneously` you mean in separate processes or just multiple urls?

**Reply by nampoinaandriamalala** [reply]
sorry, i just forget to save php file. 
It's run! TY

![image](https://user-images.githubusercontent.com/38759562/147348380-b5d1a3e5-c4a0-4e02-9c16-d2070ce708c2.png)

**Reply by nampoinaandriamalala** [reply]
separate processes
